Machine_Learning/Statistics_for_Machine_Learning/ML 1.8: A\B Testing (V).md

```diff
@@ -0,0 +1 @@
+[A/B testing Video](https://drive.google.com/file/d/1g3TtxZc80Qosk0YPz9HUdJ6E9pcRgAgv/view?usp=sharing)
```

Comment:
@karma659 Please add the thumbnail like others have done. Please do the needful changes.